src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -0,0 +1,200 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Feature extractor class for LayoutLMv2.
+"""
+
+from typing import List, Optional, Union
+
+import numpy as np
+from PIL import Image
```

Comment:
This should be under a `if is_vision_available():`

Reply:
I see that for ViT, DeiT, BEiT and DETR, this isn't done either.